_data/wildfly-categories.yaml

```diff
@@ -0,0 +1,140 @@
+---
+categories:
+  - name: Agroal
+    id: agroal
+    description: Agroal Data Source
+  - name: Batch
+    id: batch
+    description: Jakarta EE Batch
+  - name: Bean Validation
+    id: bean-validation
+    description: Jakarta EE Bean Validation
+  - name: Bootable Jar
+    id: bootable-jar
+    description: Bootable Jar
+  - name: Command Line Interface (CLI)
+    id: cli
+    description: Command Line Interface
+  - name: Cloud
+    id: cloud
+    description: Developing and Running a WildFly application on the cloud
+  - name: User Experience
+    id: user-experience
+    description: Impact how the user experiences working with WildFly
+  - name: Clustering
+    id: clustering
+    description: Clustering, including Infinispan and JGroups
+  - name: Concurrency
+    id: concurrency
+    description: Concurrency
+  - name: Console
+    id: console
+    description: Console
+  - name: WildFly Controller
+    id: controller
+    description: WildFly Controller
+  - name: WildFly Core
+    id: core
+    description: WildFly Core
+  - name: Jakarta EE
+    id: ee
+    description: Jakarta EE
+  - name: Jakarta EJB
+    id: ejb
+    description: Jakarta Entreprise Java Beans (EJB)
+  - name: WildFly Elytron
+    id: elytron
+    description: WildFly Elytron
+  - name: Embedded
+    id: embedded
+    description: Embedded
+  - name: IIOP
+    id: iiop
+    description: Internet InterORB Protocol
+  - name: Jakarta RS
+    id: jaxrs
+    description: Jakarta RESTful Web Services
+  - name: JCA
+    id: jca
+    description: Jakarta Connectors Architecture
+  - name: JDR
+    id: jdr
+    description: JBoss Diagnostic Reporter
+  - name: JPA
+    id: jpa
+    description: Jakarta Persistence
+  - name: JSF
+    id: jsf
+    description: Jakarta Faces
+  - name: Logging
+    id: logging
+    description: Logging
+  - name: Jakarta Mail
+    id: mail
+    description: Mail
+  - name: Management
+    id: management
+    description: Management
+  - name: Maven
+    id: maven
+    description: Maven Tooling & Ecosystem
+  - name: Messaging
+    id: messaging
+    description: Messaging, including Jakarta Messaging (JMS)
+  - name: Eclipse MicroProfile
```

Comment:
MicroProfile

Reply:
I made the change but I was still under the impression that it was "Eclipse MicroProfile" the first time it is mentioned?

Reply:
We thought that for a period. We may have never been right or it changed, but it's not wanted now. At some point I had some interaction with Tanja Obradovic from Eclipse where I was asking her about some WildFly publicity that used "Eclipse MicroProfile" and she asked that Eclipse be removed.

On https://microprofile.io itself the only use of "Eclipse" is the foundation etc information in the footer.